docker-compose.yml

```diff
@@ -7,7 +7,7 @@ services:
     stdin_open: true
     working_dir: /home/package
     environment:
-      - MEILISEARCH_HOST=http://meilisearch:7700
+      - MEILISEARCH_URL=http://meilisearch:7700
```

Comment:
You still have to use the new env var, otherwise, the test will not load the configuration.

Reply:
Not really. The problem is this: https://github.com/meilisearch/meilisearch-php/blob/4ed1dca16e936caac9ac8e087d9bb794a436a793/tests/TestCase.php#L50 You have to change in the TestCase.php as well :)

Reply:
Updated, thank you.